packages/bff/src/model/domain/errors.ts

```diff
@@ -43,3 +51,48 @@ export function purposeNotFound(purposeId: PurposeId): ApiError<ErrorCodes> {
     title: "Purpose not found",
   });
 }
+
+export function eServiceNotFound(eserviceId: EServiceId): ApiError<ErrorCodes> {
```

Comment:
In other PRs we discussed with @Viktor-K that it's not adding value to have branded types as input to BFF errors, because in the end we just call `unsafeBrand` when calling the error, so all these might as well be strings (and in other errors too). What do you think? So we avoid unneeded calls to unsafeBrandId